src/models/guildSettings/GuildSettings.ts

```diff
@@ -7,7 +7,11 @@ const GuildSettingsSchema = new Schema(
     guildId: { type: String, unique: true, index: true, required: true },
     adminChannelId: { type: String },
     logChannelId: { type: String },
-    pointPeriod: { type: String },
+    infoChannelId: { type: String },
```

Comment:
Şema yapısı dikey değil paralel şekilde genişlemeli. 
channels: [{name: "admin", Id: "1234" }, {name: "info", Id: "2345"} ...}

```suggestion
    channels: [ {name: { type: String }, id: { type: String }}]
```